src/rgw/rgw_multi.h

```diff
@@ -60,12 +60,18 @@ class RGWMultiXMLParser : public RGWXMLParser
 class MultipartMetaFilter : public RGWAccessListFilter {
```

Comment:
this could use a docstring (the whole filter)

Reply:
Good idea! Since the member function was non-trivial I moved it from the .h to the corresponding .cc.